src/component/registry/bpmn-elements-registry.ts

```diff
@@ -178,14 +165,19 @@ export class BpmnElementsRegistry {
    * @param classNames The name of the class(es) to remove from the BPMN element(s)
    */
   toggleCssClasses(bpmnElementIds: string | string[], classNames: string | string[]): void {
-    // TODO duplication with addCssClasses (only the call to removeClassNames or addClassNames or toggleClasses differ)
+    this.updateCssClasses(bpmnElementIds, classNames, this.cssRegistry.toggleClasses.bind(this.cssRegistry));
+  }
+
+  private updateCssClasses(bpmnElementIds: string | string[], classNames: string | string[], callback: (bpmnElementId: string, classNames: string[]) => boolean): void {
```

Comment:
❓ can we use a more explicit name for the existing `callback` parameter?
Something like `cssClassesUpdaterFunction` or `cssClassesUpdater`?

Reply:
sure, good point:)